src/Components/Quiz/CurrentQuestion.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { useQuiz } from '../../context/quizContext';
+import { Quiz } from '../../database/database.type';
+
+type Prop = {
+	currentQuiz: Quiz;
+};
+
+export const CurrentQuestion = ({ currentQuiz }: Prop) => {
+	const navigate = useNavigate();
+	const { quizState, quizDispatch } = useQuiz();
+	const [disableButtons, setDisableButtons] = useState<boolean>(false);
+	const [optionId, setOptionId] = useState<string>('');
+	const nextQuestion = () => {
+		if (currentQuiz) {
+			quizState.currentQuestionNumber === currentQuiz.questions.length - 1
+				? quizDispatch({ type: 'INCREMENT_QUESTION_NUMBER', payload: -1 })
+				: quizDispatch({ type: 'INCREMENT_QUESTION_NUMBER' });
```

Comment:
This seems a little confusing, as both actions have the same `type`, only the payload is different. 
My Suggestion here is to keep the functionality in 2 different `type` for better readability.

Reply:
yes changed